run-directive/action.yml

```
@@ -0,0 +1,54 @@
+name: Run Directive
+run-name: test
+
+inputs:
+  hash:
+    description: 'Github commit hash to checkout'
+    required: false
+    default: ''
+  working-directory:
+    description: 'Name of component directory'
+    required: true
+  config:
+    description: 'Config JSON file'
+    required: true
+  stage:
+    description: 'Name of stage'
+    required: true
+  directive:
+    description: 'A directive to run. e.g. npm ci, npm run test'
+    required: true
+
+jobs:
+  run-directive:
+    name: Run Directive
+    using: "composite"
+
+    steps:
+      - name: Setup Environment
+        uses: Brightspace/bdp-ci-action/setup-env@master
+
+      - name: Checkout branch
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ inputs.hash }}
+          fetch-depth: 0
+
+      - name: Use Node.js
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version-file: .nvmrc
+
+      - name: Add CodeArtifact npm registry
+        uses: Brightspace/codeartifact-actions/npm/add-registry@main
+        with:
+          auth-token: ${{secrets.CODEARTIFACT_AUTH_TOKEN}}
+
+      - name: Setup working directory
+        run: |
+          cp ${{ inputs.config }} ${{ inputs.working-directory }}/config.json
+          cd ${{ inputs.working-directory }}
+
+      - name: Run directive
+        run: |
```

If we leverage [`CONFIG_PATH`](https://github.com/Brightspace/bdp-configs-utils/blob/f589e61a10218a341e82b736eba0277091fd7cf8/index.js#L6) and [`working-directory`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory), I think we can remove this step? E.g.

```suggestion
      - name: Run directive
        working-directory: ${{ inputs.working-directory }}
        env:
          CONFIG_PATH: ${{ inputs.config }}
        run: |
```